database/src/main/dataset_schema/add_schema.sql

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+DROP FUNCTION IF EXISTS dataset_schema.add_schema(TEXT, INTEGER, TEXT, JSONB, TEXT);
+
+CREATE OR REPLACE FUNCTION dataset_schema.add_schema(
```

Comment:
Why dropping the function when you later offer to replace it anyway? (wouldn't either DROP+CREATE or just CREATE or REPLACE suffice?)

Reply:
You are right. After the `DROP` `CREATE` would be enough. I can change that for sure.
(My template is set up with `CREATE or REPLACE`, but unfortunately it cannot handle if the `OUT` parameters change - I think it used to have.)